app/mailers/publisher_mailer.rb

```diff
@@ -66,6 +66,17 @@ def welcome_internal(publisher)
   def verify_email(publisher)
     @publisher = publisher
     @private_reauth_url = generate_publisher_private_reauth_url(@publisher)
+
+    if @publisher.pending_email.blank?
+      begin
+        raise "SMTP To address must not be blank for PublisherMailer#verify_email"
+      rescue => e
+        require 'sentry-raven'
+        Raven.capture_exception(e,
+                                publisher: @publisher,
```

Comment:
minor: this will probably work, but it does rely on downstream serialization magic. once it lands, we can check sentry to confirm it's ok.